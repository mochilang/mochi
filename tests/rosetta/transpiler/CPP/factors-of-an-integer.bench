{
  "duration_us": 2059733,
  "memory_bytes": 3710976,
  "name": "main"
}