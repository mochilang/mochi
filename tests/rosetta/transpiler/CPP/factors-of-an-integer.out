Factors of -1 not computed

Factors of 0 not computed

Factors of 1: 
[1]
Number of factors = 1

Factors of 2: 
[1]
Number of factors = 1

Factors of 3: 
[1]
Number of factors = 1

Factors of 53: 
[1]
Number of factors = 1

Factors of 45: 
[1]
Number of factors = 1

Factors of 64: 
[1]
Number of factors = 1

Factors of 600851475143: 
[1]
Number of factors = 1

Factors of 999999999999999989: 
[1]
Number of factors = 1